framework/include/timesteppers/IterationAdaptiveDT.h

```diff
@@ -100,7 +100,9 @@ class IterationAdaptiveDT : public TimeStepper, public PostprocessorInterface
   /// cut the timestep by by this factor
   const Real & _cutback_factor;
 
-  /// Number of nonlinear iterations in previous solve
+  /// Number of solver iterations in previous solve. Linear systems contribute both to _l_its and _solver_its
```

Comment:
I think even more doxygen here would be good. Linear iteration within KSPSolves of an outer SNESSolve is still some kind of solver iteration

Reply:
I see in the docstring for `optimal_iterations` you use `outer iterations`. I like that name better, so my suggestion would be `solver_its` -> `outer_its`